src/Entities/Player.cpp

```diff
@@ -1734,70 +1701,30 @@ void cPlayer::TossPickup(const cItem & a_Item)
 
 
 
-bool cPlayer::LoadFromDisk(cWorldPtr & a_World)
+void cPlayer::LoadFromDisk(cWorldPtr & a_World)
 {
 	LoadRank();
 
-	// Load from the UUID file:
-	if (LoadFromFile(GetUUIDFileName(m_UUID), a_World))
-	{
-		return true;
-	}
+	const auto & UUID = GetUUID();
 
-	// Check for old offline UUID filename, if it exists migrate to new filename
-	cUUID OfflineUUID = cClientHandle::GenerateOfflineUUID(GetName());
-	auto OldFilename = GetUUIDFileName(GetOldStyleOfflineUUID(GetName()));
-	auto NewFilename = GetUUIDFileName(m_UUID);
-	// Only move if there isn't already a new file
-	if (!cFile::IsFile(NewFilename) && cFile::IsFile(OldFilename))
+	// Load from the UUID file:
+	if (LoadFromFile(GetUUIDFileName(UUID), a_World))
```

Comment:
It would feel more natural like this
```suggestion
	if (!LoadFromFile(GetUUIDFileName(UUID), a_World))
```

Reply:
Forget it- me dumb